cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/Extensions.kt

```diff
@@ -609,3 +642,17 @@ val ArraySubscriptionExpression.arraySize: Expression
         (((this.arrayExpression as DeclaredReferenceExpression).refersTo as VariableDeclaration)
                 .initializer as ArrayCreationExpression)
             .dimensions[0]
+
+/**
+ * This helper function calculates the "distance", i.e., number of EOG edges between this node and
+ * the node specified in [to].
+ */
+private fun Node.eogDistanceTo(to: Node): Int {
+    var i = 0
+    this.followNextEOG {
```

Comment:
Is this a shortes path distance?

Reply:
Pretty much, although its very basic and does not account for any sort of branching